style.css

```diff
@@ -50,23 +50,55 @@ li a:hover:not(.active) {
   background-color: #04AA6D;
 }
 
-img#lens {
-  max-width: 250px;
-  margin: 0 auto;
+figure {
+  display: table;
```

Comment:
This display mode on the figure will force the caption text to wrap at the width of the image.